kie-wb-smoke-tests/kie-wb-smoke-tests-main/src/main/java/org/kie/smoke/wb/util/RestRepositoryDeploymentUtil.java

```diff
@@ -90,7 +90,13 @@ public void createRepositoryAndDeployProject(String repoUrl, String repositoryNa
         
         JobRequest createRepoJob = createRepository(repositoryName, orgUnitName, repoUrl);
         waitForJobsToFinish(sleepSecs, createRepoJob);
-        
+
+        // Extra wait for indexing to finish
+        try {
+            Thread.sleep(10000); // TODO don't use hardcoded wait, but rather polling
```

Comment:
this type of addition (Thread.sleep) should not be added into standard build with (unit) tests
see our discussions about excluding any test which takes more than several seconds and includes a waiting for setup/init/finishing other steps

Tagging @jhrcek, @ge0ffrey